### PR TITLE
Add 2 icons + Link 1 icon

### PR DESCRIPTION
**Pull request by SagarDevAchar**
# Description

Added 2 icons
Linked 1 icon

### Icons added
* Ather (`com.athermobileapp`)
* MGit (`com.manichord.mg`)

### Icons linked
* Syncthing (linked `com.github.catfriend1.syncthingandroid.debug` to `@drawable/syncthing`)

## Contributor's checklist
- [x] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/.github/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

File: svgs/ather.svg
Comment (x9136) [inline svgs/ather.svg:1]: Screenshot 2023-10-16 at 10 59 45

Comment (x9136) [reply svgs/ather.svg:1]: Please increase the icon size to 160×160px (either X×160px or 160×Xpx).

**Review by x9136 — CHANGES_REQUESTED** [review]

File: svgs/ather.svg
Comment (SagarDevAchar) [reply svgs/ather.svg:1]: The smaller design is on purpose as it suits the brand design.
I had originally made it 160x160 but the outcome wasn't "nice" to look at when deployed

image

Comment (x9136) [reply svgs/ather.svg:1]: > The smaller design is on purpose as it suits the brand design.

IMG_20231016_125211
Icon size is one of the main requirements that allows the icons to display properly on the phone screen and be in the same style. Your version will be too small in relation to other icons. It's high quality, but it doesn't fit.

Screenshot 2023-10-16 at 12 45 40
The overall challenge comes down to using Lawnicons style while keeping the icon recognizable to people. There are a few variations on how to do this. I've made a couple, though there are more.

Comment (SagarDevAchar) [reply svgs/ather.svg:1]: Got it. Updated!

**Review by x9136 — APPROVED** [review]

**Pull request merged**